target_mysql/connector.py

```diff
@@ -196,13 +196,22 @@ def pick_individual_type(jsonschema_type: dict) -> sqlalchemy.types.TypeEngine:
         Returns:
             An instance of the appropriate SQL type class based on jsonschema_type.
         """
+        picked_type = None
         if "null" in jsonschema_type["type"]:
-            return None
-        if "integer" in jsonschema_type["type"]:
-            return BIGINT()
-        if jsonschema_type.get("format") == "date-time":
-            return TIMESTAMP()
-        return th.to_sql_type(jsonschema_type)
+            pass
+        elif "integer" in jsonschema_type["type"]:
+            picked_type = BIGINT()
+        elif "number" in jsonschema_type["type"]:
+            # (65,30) is the maximum precision and scale.
+            # https://dev.mysql.com/doc/refman/8.0/en/precision-math-decimal-characteristics.html
+            picked_type = DECIMAL(65, 30)
```

Comment:
65-30 = 35 digits left of the decimal point. 35 / 9 = 3.888 which means 4*4 = 16 bytes for the left of the decimal. 30 / 9 = 3.333 so 4 * 4 = 16 bytes to the right so 32 bytes per decimal character which is very large. I'm fine with leaving this but it is a consideration to make as this is a pretty large column type. Seems fine though

Reply:
Alternatives are to either pick an arbitrary smaller value which I don't like because it's a point of failure that doesn't need to be there, or allow it to be configurable. I think this is the best solution but good to keep in mind that its not the only one.